src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -48,11 +52,183 @@ public string SectionStatus(int sequenceId, int sectionId)
             return RoatpTaskListWorkflowService.SectionStatus(ApplicationSequences, NotRequiredOverrides, sequenceId, sectionId, ApplicationRouteId);
         }
 
+        public string WhosInControlCss
+        {
+            get
+            {
+                var status = WhosInControlSectionStatus;
+
+                if (status == String.Empty)
+                {
+                    return "hidden";
+                }
+
+                var cssClass = status.ToLower();
```

Comment:
why does it need to be changed to lowercase?

Reply:
This is because the CSS class names are lowercase, per convention, and are case-sensitive